src/foam/core/Property.js

```diff
@@ -293,6 +293,11 @@ foam.CLASS({
       name: 'sortable',
       value: true
     },
+    {
+      class: 'Boolean',
+      name: 'sheetsOutput',
+      value: false
```

Comment:
false is already the default default, so this line can be removed

Reply:
fixed